index/src/subrpc/mod.rs

```diff
@@ -0,0 +1,414 @@
+//! Substrate json RPC module with limited functionalites
```

Comment:
Is the code below the subrpc folder is ported from other places? If it is we can put a link here. Ignore if all are implemented by ourselves